packages/react-components/react-utilities-compat/library/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "importHelpers": true,
     "jsx": "react",
     "noUnusedLocals": true,
-    "preserveConstEnums": true
+    "preserveConstEnums": true,
+    "experimentalDecorators": true
```

Comment:
I don't see any decorator being used here , why adding this ?

Reply:
The @deprecated wasn't working initially so I added this to see, but it was my ts lang server in vscode had to be restarted.  I'll remove.